### DISCUSSION TITLE
Support for customized built-in elements

### DISCUSSION THREAD

**Discussion by K1DV5** [solidjs/solid#754] [Ideas]
I wanted to use a [customized built-in element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements#customized_built-in_elements) inside a SolidJS project, but it's not working. I'm thinking this is relevant here:
> Customized built-in elements inherit from basic HTML elements. To create one of these, you have to specify which element they extend (as implied in the examples above), and they are used by writing out the basic element but specifying the name of the custom element in the is attribute (or property). For example `<p is="word-count">`, or __`document.createElement("p", { is: "word-count" })`__

I'm guessing Solid doesn't pass the second argument to `createElement`. I think it would be nice if using these was supported.

**Comment by ryansolid** [comment]
Ok so to clarify this isn't one you want to author with Solid Element but using an existent webcomponent from else. There is a way to do it with Solid Element but it's a bit complicated. But assuming that isn't what you are asking for.

We don't do `document.createElement` calls but actually clone the nodes so I'm surprised this would be an issue. And I've used Solid with these elements before (https://github.com/ryansolid/solid-hackernews-app/blob/master/src/elements/AppNav/index.js#L8)

For example:
```js
<p is="word-count"/>
```
Compiles to roughly:
```js
const template = document.createElement("template")
template.innerHTML = `<p is="word-count"/>`
const p = template.content.firstChild.cloneNode(true);
```
You can see what I mean here by looking at the output tab: https://playground.solidjs.com/?hash=-1653681417&version=1.2.5

Maybe a reproduction would help.
